chapters/data/memory-security/drills/tasks/exec-shellcode/support/tests/Makefile

```
@@ -1,5 +1,5 @@
 SRC_PATH ?= ../src
-FULL_SRC_PATH = $(realpath $(SRC_PATH))
+FULL_SRC_PATH = "$(realpath $(SRC_PATH))"
```

```suggestion
FULL_SRC_PATH = "$(realpath "$(SRC_PATH)")"
```
I think you also need these to avoid `realpath` failling on spaced paths.

---

No reason to add the inner set of quotes. `realpath` works with paths that contain spaces. You only need to quote the result.